powerline/segments/i3wm.py

```diff
@@ -16,12 +16,12 @@ def calcgrp(w):
 	return group
 
 
-def workspaces(pl):
+def workspaces(pl, strip=0):
 	'''Return workspace list
 
 	Highlight groups used: ``workspace``, ``w_visible``, ``w_focused``, ``w_urgent``
 	'''
 	return [{
-		'contents': w['name'],
+		'contents': w['name'][min(len(w['name']),strip):],
```

Comment:
the stripping is there because i3 has a convention where you prepend numbers to your workspaces so that they stay in order ("1: www", "2: comm" etc.). I accidentally overwrote my old version and quickly recovered it like this (which is why I forgot to add documentation for `strip` again), but there probably is a smarter way.

I am not sure whether a format string is too fitting here? A regex that will be replaced with nothing would be more useful maybe (`"[0-9]+: "` in the config would strip numbers. colon and space wherever they exist)?